server/lists/adapters/workstacks.js

```diff
@@ -125,23 +125,24 @@ const highestPriorityContributionStatus = (decoratedContributions) => {
     let highestPriority = 0;
 
     decoratedContributions.map(decoratedContribution => {
```

Comment:
This should probably be a `forEach`, map creates a new array and is for the transformation of data.

Reply:
Could also use `reduce` if you are feeling fancy...

Reply:
Agreed, gone for forEach, apologies for reduced fanciness levels